lib/generators/hyrax/config_generator.rb

```diff
@@ -38,6 +39,14 @@ def configure_redis
     copy_file 'config/initializers/redis_config.rb'
   end
 
+  # @note We added these to address the fact that the SOLR schema we
+  # use broke in the SOLR 8.6.3 release.  We envision this to be a
+  # stop-gap.
+  def configure_solr_wrappers
+    copy_file 'config/solr_wrapper_test.yml'
```

Comment:
Looks like the build failed due to timing out waiting for user input on copying the solr_wrapper test config since it was already generated by `active_fedora:config:solr`.  So maybe it is best to just force overwriting:
```suggestion
    copy_file 'config/solr_wrapper_test.yml', force: true
```